test/sync/src/androidTest/kotlin/io/realm/test/mongodb/shared/SyncSessionTests.kt

```diff
@@ -87,6 +88,25 @@ class SyncSessionTests {
         }
     }
 
+    @Test
+    fun sessionPauseAndResume() {
```

Comment:
There are a few more tests in https://github.com/realm/realm-java/blob/f0eca830527cd8255ce8fb9bcc83a545a940f6d3/realm/realm-library/src/syncIntegrationTest/kotlin/io/realm/SyncSessionTests.kt#L729 we should probably move as well (i.e. calling pause/resume multiple times)